tests/module/memory_test.c

```diff
@@ -0,0 +1,191 @@
+
+#include "VecSim/memory/redismodule.h"
+#include "VecSim/vec_sim.h"
+#include <stdbool.h>
+#include <errno.h>
+#include <string.h>
+
+#define DIMENSION 100
+
+// get memory usage from redis-server
```

Comment:
note here: a comment sentence starts with a capital letter and ends with a period. please apply it to the entire file.